root.spec

```diff
@@ -146,7 +163,43 @@ done
 export ROOT_INCLUDE_PATH
 
 export ROOTSYS=%i
+
 make INSTALL="$cp" INSTALLDATA="$cp" install
+# Handle debug symbols. We always have a debug subpackage for debug symbols so
+# no point in complicating the standard builds with this.
+
+%if "%{?subpackageDebug:set}" == "set"
+  ELF_DIRS="%i/lib %i/bin"
+  DROP_SYMBOLS_DIRS=""
+
+# optimise the debug symbols, compress them, and split them into separate file
```

Comment:
What is the size of installtion (i.e. not compressed)?
Check size of ROOT installation before and after debug was introduced (that should not have significant increase in size). Then also check how much that increases after debug package is installed.